src/Dennis/Link/Checker/Logger.php

```diff
@@ -145,6 +145,7 @@ public function critical($message, array $context = array()) {
    */
   public function error($message, array $context = array()) {
     $this->addRecord(self::ERROR, (string) $message, $context);
+    watchdog('dennis_link_checker_seo', $message);
```

Comment:
Might this be a good opportunity to add WATCHDOG_ERROR/NOTICE/INFO log levels while we're here?

E.g. `watchdog('dennis_link_checker_seo', $message, [], WATCHDOG_ERROR);`